code/scripting/api/objs/sound.cpp

```diff
@@ -124,6 +124,15 @@ ADE_FUNC(isValid, l_SoundEntry, NULL, "Detects whether handle is valid", "boolea
 	return ade_set_args(L, "b", seh->IsValid());
 }
 
+ADE_FUNC(exists, l_SoundEntry, nullptr, "Detects whether handle references a sound that can be loaded", "boolean", "true if exists, false if not, nil if a syntax/type error occurs")
```

Comment:
If we change the function name as requested above, we should change the name here as well.

Reply:
Done